src/Xamarin.CodeAnalysis/PListObject.cs

```diff
@@ -0,0 +1,2657 @@
+﻿// 
+// PObject.cs
+//  
```

Comment:
TODO for this file: remove `POBJECT_MONOMAC` sections and `POBJECT_INTERNAL` as appropriate.

Reply:
There are likely other chunks of code that are not part of the limited plist parser functionality that the `AttributeAnalyzer` uses. These should be removed as well.